pom.xml

```diff
@@ -94,6 +94,9 @@
                 <version>${javafx.plugin.version}</version>
                 <configuration>
                     <mainClass>${main.class}</mainClass>
+                    <options>
+                        <option>--add-exports=javafx.graphics/com.sun.javafx.tk=ALL-UNNAMED</option>
```

Comment:
Could this please be turned into a motivation to move some of that API into the public area?

Reply:
Right, but you know how it goes, making public exiting private API is challenging to say the least...
For instance, demand for public `FontMetrics` dates back to 2010...  https://bugs.openjdk.java.net/browse/JDK-8090775
The last comment about using the old com/sun/javafx/scene/control/skin/Utils.java gave me an idea, and I've been able to remove the private API...